### ISSUE TITLE
[Play Games Plugin DLL] ERROR: Exception launching token request

### ISSUE THREAD

**Issue by maciejorlinski** [playgameservices/play-games-plugin-for-unity#1906]
I get a following error after I try authenticating user in game:

```
*** [Play Games Plugin DLL] 09/05/17 17:06:37 +02:00 ERROR: Exception launching token request: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment
                                          UnityEngine.DebugLogHandler:Internal_Log(LogType, String, Object)
                                          UnityEngine.DebugLogHandler:LogFormat(LogType, Object, String, Object[])
                                          UnityEngine.Logger:Log(LogType, Object)
                                          UnityEngine.Debug:LogWarning(Object)
                                          GooglePlayGames.OurUtils.<e>c__AnonStorey2:<>m__0() (at D:\quizy-unity\Quizy\Assets\GooglePlayGames\OurUtils\Logger.cs:77)
                                          GooglePlayGames.OurUtils.PlayGamesHelperObject:Update() (at D:\quizy-unity\Quizy\Assets\GooglePlayGames\OurUtils\PlayGamesHelperObject.cs:135)
                                           
                                          (Filename: D Line: 0)
09-05 17:06:37.857 22235-22302/? W/Unity: *** [Play Games Plugin DLL] 09/05/17 17:06:37 +02:00 ERROR: UnityEngine.AndroidJavaException: java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment
                                          java.lang.ClassNotFoundException: com.google.games.bridge.TokenFragment
                                              at java.lang.Class.classForName(Native Method)
                                              at java.lang.Class.forName(Class.java:324)
                                              at java.lang.Class.forName(Class.java:285)
                                              at com.unity3d.player.UnityPlayer.nativeRender(Native Method)
                                              at com.unity3d.player.UnityPlayer.a(Unknown Source)
                                              at com.unity3d.player.UnityPlayer$b$1.handleMessage(Unknown Source)
                                              at android.os.Handler.dispatchMessage(Handler.java:98)
                                              at android.os.Looper.loop(Looper.java:148)
                                              at com.unity3d.player.UnityPlayer$b.run(Unknown Source)
                                           Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.games.bridge.TokenFragment" on path: DexPathList[[zip file "/data/app/pl.agora.quizy-1/base.apk"],nativeLibraryDirectories=[/data/app/pl.agora.quizy-1/lib/arm, /data/app/pl.agora.quizy-1/base.apk!/lib/armeabi-v7a, /vendor/lib, /syst
```

I think it started appearing after play services updated to 11.0.2. I tried doing things other people suggested when they got this error but nothing helped so far.

**Comment by claywilkinson** [comment]
Which version of Unity are you using?  Re-run the play services jar resolver?  If you are still having problems can you share the list of files in Assets/Plugins/Android?

**Comment by maciejorlinski** [comment]
@claywilkinson yeah, after play services resolved to 11.0.4, the problem is gone. Thanks!

**Comment by claywilkinson** [comment]
Great!
